### ISSUE TITLE
Comprehensive Test Plan: Dump the cypress_accessibility_violations.json file into the PR after cypress tests have run in tugboat

### ISSUE THREAD

**Issue by keisterj-oddball** [department-of-veterans-affairs/va.gov-cms#19412]
## User Story or Problem Statement
As an engineer working within the CMS, I need the contents of the `cypress_accessibility_violations.json` available within my PR, so that I am able to easily review any newly reported accessibility violations reported during cypress testing.

## Description or Additional Context 
This file is written on the file system within tugboat, but is currently only accessible via terminal in tugboat.  This makes reviewing the reported issues tedious and time consuming.

A consideration is that tugboat content will disappear. 

## Steps for Implementation
- Explore if the results should be linked versus content dumped in PR as a comment

## Acceptance Criteria
- [ ] The contents of the violations report should be directly accessible via the PR in github
- [ ] The contents of the violations report should match between github and tugboat
- [ ] Accessibility violations should be highlighted in PRs but not explicitly fail cypress tests

**Comment by gracekretschmer-metrostar** [comment]
Possibly update this ticket to include changing the accessibility errors within the tests.
